_pkg.dev/crypto/privatekey/privatekey.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/CityOfZion/neo-go/pkg/crypto/base58"
 	"github.com/CityOfZion/neo-go/pkg/crypto/elliptic"
 	"github.com/CityOfZion/neo-go/pkg/crypto/hash"
-	"github.com/CityOfZion/neo-go/pkg/crypto/rfc6979"
+	"github.com/anthdm/rfc6979"
```

Comment:
I believe this library is smth we wanted to dive deeper later, right?

Reply:
Not exactly, at least until there are some issues found there. It seems to work and at the same time it doesn't seem to be actively maintained by anybody, all the forks just have the original five years old code, it's just that for us it's better not to have a copy of its source code inside (as per #285).

Reply:
we can't just use ecdsa.Sign / Verify?